helm-chart/templates/nextcloud-aio-talk-service.yaml

```diff
@@ -19,4 +21,21 @@ spec:
       targetPort: {{ .Values.TALK_PORT }}
   selector:
     io.kompose.service: nextcloud-aio-talk
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c -f latest.yml
+    kompose.version: 1.28.0 (c4137012e)
+  labels:
+    io.kompose.service: nextcloud-aio-talk
+  name: nextcloud-aio-talk
+spec:
+  ports:
+    - name: "8081"
+      port: 8081
+      targetPort: 8081
+  selector:
+    io.kompose.service: nextcloud-aio-talk
```

Comment:
@st3iny does this look good to you now? :)

I only wonder if it really works since basically there are now two services with the name nextcloud-aio-talk. So should I just rename this one? Also do you know if it is fine if there are multiple services with label and selector `io.kompose.service: nextcloud-aio-talk`?

Reply:
Ah I just read that the dns will use the given service name. So using a different name is also not a valid approach because then communication would need to take place using this different hostname? I guess just adding the port to the loadbalancer is the best approach then? Like as it was before? I mean yij should still be able to configure the firewall to not expose this internal port, no?

Reply:
ah, I think I found solution: only the internal name needs to work since the loadbalancer should be publicly rechable using its doman. So I can just rename the loadbalancer.